.github/workflows/evm-version-compatibility-tests.yml

```diff
@@ -55,11 +54,75 @@ jobs:
+      latest-ethereum-client-image-exists: ${{ steps.check-latest-ethereum-client-image-exists.outputs.exists }}
+    steps:
+      - name: Checkout chainlink-testing-framework repository
+        uses: actions/checkout@v2
+        with:
+          repository: smartcontractkit/chainlink-testing-framework
+          path: chainlink-testing-framework
+      - name: Set up Go
+        uses: actions/setup-go@v3
+      - name: Run get-latest-ethereum-client-version ethereum/client-go:latest_stable command
+        id: get-latest-ethereum-client-version
+        run: |
+          cd chainlink-testing-framework/
+          ls utils/cmd/
+          version=$(go run utils/cmd/main.go get-latest-ethereum-client-version ethereum/client-go:latest_stable)
+          echo "version=${version}" >> $GITHUB_OUTPUT
+      - name: Check if latest Ethereum client image ${{ steps.get-latest-ethereum-client-version.outputs.version }} exists in ECR
+        id: check-latest-ethereum-client-image-exists
+        uses: smartcontractkit/chainlink-github-actions/docker/image-exists@b49a9d04744b0237908831730f8553f26d73a94b # v2.3.17
+        with:
+          repository: ethereum/client-go
+          tag: ${{ steps.get-latest-ethereum-client-version.outputs.version }}
+          AWS_REGION: ${{ secrets.QA_AWS_REGION }}
+          AWS_ROLE_TO_ASSUME: ${{ secrets.QA_AWS_ROLE_TO_ASSUME }}
+      - name: Latest Ethereum client image does not exist in ECR
+        if: steps.check-latest-ethereum-client-image-exists.outputs.exists != 'true'
+        run: |
+          echo "ethereum/client-go:${{ steps.get-latest-ethereum-client-version.outputs.version }} does not exist in our ECR. The image download was scheduled, and you can rerun this workflow in few minutes."
+          exit 1          
+
+  # If the required images do not exist, trigger the update-internal-mirrors workflow in CTF and fail the current workflow
+  trigger-update-internal-mirrors-workflow-in-ctf:
+    name: Trigger update-internal-mirrors workflow in CTF
+    needs: [check-ecr-images-exist]
+    if: ${{ failure() && needs.check-ecr-images-exist.outputs.latest-ethereum-client-image-exists != 'true' }}
+    permissions:
+      actions: read
+      id-token: write
+      contents: read
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup GitHub Token
+        id: setup-github-token
+        uses: smartcontractkit/.github/actions/setup-github-token@9e7cc0779934cae4a9028b8588c9adb64d8ce68c # setup-github-token@0.1.2
+        with:
+          aws-role-arn: ${{ secrets.AWS_OIDC_CHAINLINK_CI_INVOKE_CTF_WORKFLOWS_TOKEN_ISSUER_ROLE_ARN }}
+          aws-lambda-url: ${{ secrets.GATI_QA_LAMBDA_URL }}
+          aws-region: ${{ secrets.AWS_REGION }}
+      - name: Trigger update-internal-mirrors workflow in CTF
+        run: gh workflow run --repo smartcontractkit/chainlink-testing-framework update-internal-mirrors.yaml
+        env:
+          GITHUB_TOKEN: ${{ steps.setup-github-token.outputs.access-token }}
+
```

Comment:
please wait here for that job to finish (can be in a loop with sleep if there's no other way) and fail the workflow only if the update job fails

Reply:
Fixed